### PR TITLE
upgrades dependencies

### PR DESCRIPTION
**Pull request by jaimeiniesta**
Hello!

This PR upgrades Elixir and dependencies to their latest versions.

### CODE REVIEW

**Comment by flowerett** [comment]
Hello everybody!
And thanks for this contribution, @jaimeiniesta !
Are there any plans to merge this MR or #20 any time soon?
Almost half of the year has passed. Also, we're successfully using these changes in some projects in production.
cc: @JuneKelly

File: mix.exs [hunk 1]
```diff
@@ -48,12 +48,12 @@ defmodule HammerBackendRedis.Mixfile do
   # Type "mix help deps" for more examples and options
   defp deps do
     [
-      {:redix, "~> 0.8"},
+      {:redix, "~> 1.0"},
       {:hammer, "6.0.0"},
```
Comment (longnd) [inline mix.exs:52]: Should we also update `hammer` as new version ([6.1](https://github.com/ExHammer/hammer/blob/master/CHANGELOG.md#610---2022-06-13)) released?

**Review by longnd — COMMENTED** [review]
hi @epinault, would you mind review and merge this PR 🙏

File: mix.exs (hunk 1 above)
Comment (jaimeiniesta) [reply mix.exs]: hi @longnd sure, I've updated this dependency as well.

File: .tool-versions [hunk 2]
```diff
@@ -1 +1 @@
-elixir 1.6
+elixir 1.12.0-otp-23
```
Comment (epinault) [inline .tool-versions:1]: might as well put 1.13?

**Comment by epinault** [comment]
done with 6.1.1

**Pull request closed without merging**